tests/workflow/test_jobs.py

```diff
@@ -163,3 +163,37 @@ def test_run_with_profile():
         assert (
             f"Profiling [{project.unified_dataset().name}] attributes." == all_ops[1].description
         )
+
+
+@mock_api()
+def test_get_project_output_datasets():
```

Comment:
This test looks good to me, but it might be interesting to parametrize it.  Right now you are testing 4 cases in a single test run, but with a little re-writing you could make the test run four times on different inputs and check for different expected outputs.
This is done with the `@pytest.mark.parametrize` decorator, e.g.:
```
@pytest.mark.parametrize(
    "var_a,var_b,var_c",
    [
        (case_1_var_a, case_1_var_b, case_1_var_c),
        (case_2_var_a, case_2_var_b, case_2_var_c),
        (case_3_var_a, case_3_var_b, case_3_var_c)
    ]
)
def my_test_function(var_a, var_b, var_c):
    assert my_function(var_a, var_b) == var_c
```

Reply:
Sure thing, @skalish - this seems pretty slick - that said, I will need to have two tests since for SM and Enrichment types I am making a direct dataset name comparison but for Mastering, GR, and Categorization types I am basing the test on the number of outputs retrieved.